lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -398,8 +398,21 @@ def start(self):
         migration_bins = self.data_bin.energy_migration_bins()
         source_offset_bins = self.data_bin.source_offset_bins()
 
+        if self.mc_particle["gamma"]["mc_type"] in ["point_like", "ring_wobble"]:
+            # The 4 is semi-arbitray. This keeps the same precision as the previous code
+            mean_fov_offset = np.round(get_mc_fov_offset(self.mc_particle["gamma"]["file"]), 4)
+            self.log.info(f"Single offset for point like gamma MC with offset {mean_fov_offset}")
+            fov_offset_bins = [mean_fov_offset - 0.1, mean_fov_offset + 0.1] * u.deg
+        else:
+            fov_offset_bins = self.data_bin.fov_offset_bins()
+            self.log.info("Multiple offset for diffuse gamma MC")
```

Comment:
Perhaps you can print in the log the binning (might be somewhere else, but better be explicit on what it is doing)

Reply:
I've added in the log line the FoV binning used.